R/utils.R

```diff
@@ -224,6 +224,23 @@ platform_independent_sort <- function(x) x[platform_independent_order(x)]
 #'   extract its string value with [xml2::xml_text()]. If `s` is an `xml_node` or `xml_nodeset`
 #'   and `xpath` is specified, it is extracted with [xml2::xml_find_chr()].
 #' @param xpath An XPath, passed on to [xml2::xml_find_chr()] after wrapping with `string()`.
+#'
+#' @examples
+#' tmp <- withr::local_tempfile(lines = "c('a', 'b')")
+#' expr_as_xml <- get_source_expressions(tmp)$expressions[[1L]]$xml_parsed_content
+#' writeLines(as.character(expr_as_xml))
+#' get_r_string(expr_as_xml, "expr[2]") # "a"
+#' get_r_string(expr_as_xml, "expr[3]") # "b"
+#'
+#' # more importantly, extract strings under R>=4 raw strings
+#' if (getRversion() >= "4.0.0") {
```

Comment:
Instead of `if()` block, should we use `@examplesIf` instead?

Due to [this](https://github.com/r-lib/pkgdown/issues/2175) issue, only the output for `get_r_string(expr_as_xml4.0, "expr[3]")` will be shown on the website, but not `get_r_string(expr_as_xml4.0, "expr[2]")`.

Reply:
Good idea for simplicity, but does it actually help with printing? Since roxygen is just hiding the `if()` statement?

Reply:
I checked this with my [minimalReprex](https://github.com/IndrajeetPatil/minimalReprex) package:

# Using `if()`

<img width="739" alt="Screenshot 2022-10-08 at 07 29 51" src="https://user-images.githubusercontent.com/11330453/194690410-13fc6ae6-d08e-46b1-9d8e-37bf79f84db6.png">

# Using `@examplesIf`

<img width="611" alt="Screenshot 2022-10-08 at 07 34 11" src="https://user-images.githubusercontent.com/11330453/194690568-54bf34d1-c124-4b6e-9ff4-efcb18d5cb97.png">

So this definitely helps.

Reply:
awesome, nice find!